docs/README-AWS.md

```diff
@@ -75,35 +60,7 @@ On top of this basic permission the following policies must be provided:
 
 * For SQS queue resource that you want to use as triggers of the Lambda the proper permission are already included by `arn:aws:iam::aws:policy/service-role/AWSLambdaSQSQueueExecutionRole`:
 
```

Comment:
Either this is true or line 48. Can we combine the IAM requirement for trigger SQS queue in one place.

Reply:
Their are both true someway.

Line 48 is related to specific error handling in batch processing of sqs messages.
The most straightforward and reliable way to handle them is to delete a message once it's processed, this lead to the needs of `GetQueueUrl` permission

Another option is what's describe here: https://docs.aws.amazon.com/lambda/latest/dg/with-sqs.html#services-sqs-batchfailurereporting

In my opinion is much more fragile and requires extra configuration step for the users on their own queues.

Reply:
We might/should rephrase line 61 probably. Since it doesn't reflect 100% what's required